experimental/css-has-pseudo/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changes to CSS Has Pseudo
+
+### Unreleased
+
+Tracking initial implementation of `:has()` pseudo-class in Safari Technology Preview.
+This is a breaking change and affects both the generated CSS and the client side polyfill.
```

Comment:
Could we couple minor examples (but meaningful, not something like .foo but just an input with hover and that sort of thing to quickly identify how's changing? Just an idea, not really a request

Reply:
I added a few examples here but do not want to go into too much detail.
Given the complexity of `:has` it's difficult to explain something without explaining everything (or at least for me)